src/js/views/globals/search/patient-search.scss

```diff
@@ -47,6 +47,13 @@
   margin-left: 16px;
 }
 
+.patient-search__picklist-item-meta:empty::before {
+  color: $black-60;
+
+  // &ndash;
+  content: '–';
+}
+
```

Comment:
This will show the `-` if:

* Patient identifier doesn't exist. 
* Patient identifier exists, but its value is an empty string.

Both scenarios are assuming the given identifier is set via the `patient_search_settings` org setting.

Matches what we do for empty widget values [here](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/scss/modules/widgets.scss).